rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+
+- According to the poing above, user engagement becomes one of the most important things on a website. Which means adding the comment section is a good idea.
+
+- Keywords are important but sometimes the most authoritative keywords are not the best ones. High-volume keywords are also more competitive to rank higher (keyword difficulty). Carefully [exploring the long tail](https://moz.com/blog/long-tail-seo-target-low-volume-keywords-whiteboard-friday) of relevant keywords to us might offer us a better advantage in ranking and in financial conversions.
+
+  > For example, a person searching for "shoes" is probably just browsing. On the other hand, someone searching for "best price red womens size 7 running shoe" practically has their wallet out!
+
+- Perhaps we were not intentional enough concerning keywords in our current content. According to the point above, phrases formulated as questions are SEO gold. We could do a search on the most common questions on our space and make sure these questions are phrased in our content
+
+- Writing content is HARD so we need to make sure we are making the most for our buck. We need a strategy before deciding which content to write about. I've seen these two content strategies which seem very interesting
+
+  - 10x content (😂 geez I *hate* the name)
+    - Search the keyword(s) you want your page to rank for
+    - Identify which pages are ranking highly for those keywords
+    - Determine what qualities those pages possess
+    - Create content that’s better than that
+  - Top content refurbishment
+    - See which of your content performs better
+    - Take that content and [refurbish it for other platforms](https://moz.com/blog/refurbishing-top-content-whiteboard-friday) (Slideshare, YouTube, Instagram, Quora, etc.) to expand acquisition funnel beyond Google.
+
+- Page speed is a very important SEO metric so we need to be extremely careful when making the website dynamic. Perhaps it's best to never migrate the guides to be dynamic? But images are the usually the top 1 culprit in page speed. We should consider optmizing the images for mobile and compressing all the images we use.
+
+- There seems to be no correlation with text length and rankings (I looked this up to see if it is a problem that our guides are very long and if maybe we should break them up, but apparently it doesn't matter)
+
+- Breadcrumbs are SEO-useful if they contain [micro data](http://schema.org/breadcrumb)
+
+- Google ranks accessibility and mobile-friendliness
+
+- Trustworthiness goes a long way both for search **and for users**. I think we need to make more prominent on our website that we maintain terratest, terragrunt & that jim wrote terraform up and running.
```

Comment:
How is trustworthiness measured by Google?

Reply:
backlinking by authorithative websites (we already know about this, so I didn't go into details here), but it got me thinking that users probably have the same concerns

Reply:
A good chunk of our users come to us exactly because they found about us from these sources. But now that we are targeting more organic approaches, they might not be aware.